capabilities/istio/.github/zarf-runner/deploy/zarf.yaml

```diff
@@ -0,0 +1,30 @@
+kind: ZarfPackageConfig
+metadata:
+  name: farmer-pickles
+  url: https://welovebob.com/characters/farmer_pickles.htm
+  image: https://welovebob.com/Images/characters/farmer_pickles.jpg
+  description: "A zarf package to deploy things"
+
+components:
+  - name: deploy
+    required: true
+    actions:
+      onCreate:
+        before:
+          - description: Deploy the cluster bundle
+            cmd: |
+                ARCH=$(uname -m)
+                [ "$ARCH" = "x86_64" ] && ARCH="amd64"
+                ./uds deploy oci://ghcr.io/zachariahmiller/k3d-common:0.1.0-$ARCH --no-progress --confirm
```

Comment:
we shouldn't be pointing to personal repos

Reply:
100%. the code that deploys this oci://ghcr.io/zachariahmiller/k3d-common:0.1.0-$ARCH needs to be moved into the org at which time this will not reference my local repo.

Reply:
I can do the supported work for this ASAP, but will need to do some refactoring to make it deploy both package/ bundle architectures.